frontend/packages/console-app/src/components/cloud-shell/CloudShellTabHeader.tsx

```diff
@@ -0,0 +1,33 @@
+import * as React from 'react';
+import { Flex, FlexItem } from '@patternfly/react-core';
+// import { FlexModifiers, Button } from '@patternfly/react-core';
+// import { AngleLeftIcon } from '@patternfly/react-icons';
+import './CloudShellTab.scss';
+
+const CloudShellTabHeader = () => {
+  // const onClose = () => {
+  //   window.opener = null;
+  //   window.open('', '_self');
+  //   window.close();
+  // };
+  return (
+    <Flex style={{ flexGrow: 1 }}>
```

Comment:
Use flex modifiers prop instead of style - https://www.patternfly.org/v4/documentation/react/layouts/flex

Reply:
@rohitkrai03 This portion is same as that in the CloudShellDrawerHeader

Reply:
@abhinandan13jan What is the need to specify the style like this in both the cases? Wouldn't this work - `breakpointMods={[{modifier: FlexModifiers.grow}]}`

Reply:
yes, it works...updated..thanks